matchzoo/datasets/snli/load_data.py

```diff
@@ -10,7 +10,7 @@
 _url = "https://nlp.stanford.edu/projects/snli/snli_1.0.zip"
 
 
-def load_data(stage='train', task='ranking', target_label='entailment'):
+def load_data(stage='train', task='classification', target_label='entailment'):
```

Comment:
The default task is usually `ranking` across the entire project.

Reply:
This is dataset is originally a classification problem. We will use it to demonstrate how to use matchzoo to accomplish classification tasks.